client/src/components/ContactForm.js

```diff
@@ -35,9 +35,9 @@ export default () => {
     })
   }
   return (
-    <div className="contact-form-wrapper" id="contact-form-id">
+    <div className="contact-form-wrapper" id="Contact">
       <h1 className="contact-form-header-text">Contact Us</h1>
-      <form onSubmit={handleSubmit} id="contact-form">
+      <form onSubmit={handleSubmit} class="contact-form">
```

Comment:
Change to className, throwing error

Reply:
Hmm.  Not sure I know what you mean?  I changed it so all the ID's are no longer being used for styles.  If it was a hash that was being used for styles and the scroll to I changed it.

Reply:
I'm not getting any errors when I pull down and use the site. My guess is maybe Macattack1999-teck has something old interfering with what was pushed here.